modules/flowable-http/src/main/java/org/flowable/http/bpmn/impl/HttpActivityBehaviorImpl.java

```diff
@@ -141,84 +143,95 @@ public boolean verify(String s, SSLSession sslSession) {
-            request.setSaveResponseTransient(getBooleanFromField(saveResponseParametersTransient, execution));
-            request.setSaveResponseAsJson(getBooleanFromField(saveResponseVariableAsJson, execution));
-            request.setPrefix(getStringFromField(resultVariablePrefix, execution));
-
-            String failCodes = getStringFromField(failStatusCodes, execution);
-            String handleCodes = getStringFromField(handleStatusCodes, execution);
-
-            if (failCodes != null) {
-                request.setFailCodes(getStringSetFromField(failCodes));
-            }
-            if (handleCodes != null) {
-                request.setHandleCodes(getStringSetFromField(handleCodes));
-            }
+        String skipExpressionText = httpServiceTask.getSkipExpression();
 
-            if (request.getPrefix() == null) {
-                request.setPrefix(execution.getCurrentFlowElement().getId());
-            }
+        CommandContext commandContext = CommandContextUtil.getCommandContext();
 
-            // Save request fields
-            if (request.isSaveRequest()) {
-                execution.setVariable(request.getPrefix() + "RequestMethod", request.getMethod());
-                execution.setVariable(request.getPrefix() + "RequestUrl", request.getUrl());
-                execution.setVariable(request.getPrefix() + "RequestHeaders", request.getHeaders());
-                execution.setVariable(request.getPrefix() + "RequestBody", request.getBody());
-                execution.setVariable(request.getPrefix() + "RequestBodyEncoding", request.getBodyEncoding());
-                execution.setVariable(request.getPrefix() + "RequestTimeout", request.getTimeout());
-                execution.setVariable(request.getPrefix() + "DisallowRedirects", request.isNoRedirects());
-                execution.setVariable(request.getPrefix() + "FailStatusCodes", failCodes);
-                execution.setVariable(request.getPrefix() + "HandleStatusCodes", handleCodes);
-                execution.setVariable(request.getPrefix() + "IgnoreException", request.isIgnoreErrors());
-                execution.setVariable(request.getPrefix() + "SaveRequestVariables", request.isSaveRequest());
-                execution.setVariable(request.getPrefix() + "SaveResponseParameters", request.isSaveResponse());
-            }
+        boolean isSkipExpressionEnabled = SkipExpressionUtil.isSkipExpressionEnabled(skipExpressionText,
+                execution.getCurrentActivityId(), execution, commandContext);
 
-        } catch (Exception e) {
-            if (e instanceof FlowableException) {
-                throw (FlowableException) e;
-            } else {
-                throw new FlowableException(HTTP_TASK_REQUEST_FIELD_INVALID + " in execution " + execution.getId(), e);
+        if (skipExpressionText == null || !isSkipExpressionEnabled || !SkipExpressionUtil.shouldSkipFlowElement(skipExpressionText,
```

Comment:
skipExpressionText == null check is not necessary here, can you remove this?

Reply:
Removed redundant null check and updated test to ensure that Http Tasks without a skip expression are still executed when _FLOWABLE_SKIP_EXPRESSION_ENABLED is set.